app/javascript/courses/students/components/CoursesStudents__LevelDistribution.css

```diff
@@ -0,0 +1,42 @@
+.course-students-root__student-distribution-level-pill {
```

Comment:
Use the component's name for the _block_. For example, this class can be `level-distribution__pill`. All other classes in should also be cleaned up.

This style isn't even used in `CourseStudents__Root`, so I'm guessing this is an holdover from old styling?